### PR TITLE
alan

### PR DESCRIPTION
**Pull request by alan-zheng**
add another function to solve some troubles
branch tag v.17.2

### CODE REVIEW

**Comment by alan-zheng** [comment]
recommit agin

**Comment by alan-zheng** [comment]
no need merge

File: context.go [hunk 1]
```diff
@@ -16,3 +17,8 @@ func contextSet(r *http.Request, key, val interface{}) *http.Request {
 
 	return r.WithContext(context.WithValue(r.Context(), key, val))
 }
+
+func gotest(){
+	a := 123
+	fmt.Printf("%+10d\n", a)  //+123
+}
```
Comment (ashar154) [inline context.go]: please put new line at the end of file.

**Comment by elithrar** [comment]
I don't think you mean to PR this here.

**Pull request closed without merging**